manifests/kubernetes-mcp-server-deployment.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kubernetes-mcp-server
+  namespace: kubernetes-mcp-server
+  labels:
+    app.kubernetes.io/name: kubernetes-mcp-server
+    app.kubernetes.io/version: v0.0.51
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: kubernetes-mcp-server
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: kubernetes-mcp-server
+        app.kubernetes.io/version: v0.0.51
+    spec:
+      serviceAccountName: kubernetes-mcp-server
+      securityContext:
+        runAsNonRoot: true
+        runAsUser: 1000
+        runAsGroup: 1000
+        fsGroup: 1000
+      containers:
+      - name: kubernetes-mcp-server
+        image: quay.io/manusa/kubernetes_mcp_server:v0.0.51
+        ports:
+        - containerPort: 8080
+          name: http
+          protocol: TCP
+        args:
+        - "--port"
```

Comment:
Since we exercise oauth, I think we need a toml file with required flags -- --require-oauth, etc.

Reply:
Sure, I just assumed this was iterative work.

In case we're focusing on token exchange, then it's too soon to implement the manifests (IMO).
I'm sure we're going to change a few things once the environment is ready.

I'll come back to the manifests once I've tested the token-exchange workflow works.